### ISSUE TITLE
Experimental data: Image-text retrieval R@1 is too low

### ISSUE THREAD

**Issue by chirmy** [alinlab/s-clip#2]
Hello,Thank you for sharing such great code. But I have a question, I noticed in your paper that the Image text retrieval R@1 parameter metric used in the RSICD dataset is 4.2, but my actual result of running the program is that the Image text retrieval R@1 is only about 0.04, the following is my parameter configuration.

`METHOD=(
"ours"
"base"
)
SEED=("0")
RATIO=(
"0.1"
)
MODEL=(
"RN50"
#"ViT-B-32"
#"ViT-B-16"
) 
ImagenetVal=(
#"RSICD-CLS"
#"UCM-CLS", 
#"WHU-RS19", 
#"RSSCN7", 
#"AID", 
"RESISC45"
)

for val in "${ImagenetVal[@]}"; do
for model in "${MODEL[@]}"; do
for ratio in "${RATIO[@]}"; do
for method in "${METHOD[@]}"; do
for seed in "${SEED[@]}"; do
    torchrun --nproc_per_node 2 -m main \
        --model "${model}" \
        --pretrained openai \
        --train-data "RS-ALL" \
        --label-ratio "${ratio}" \
        --val-data "RS-ALL" \
        --imagenet-val "${val}" \
        --keyword-path "keywords/RS/class-name.txt" \
        --lr 5e-5 \
        --batch-size 64 \
        --warmup 10 \
        --epochs 25 \
        --zeroshot-frequency 3 \
        --precision amp \
        --method "${method}" \
        --seed "${seed}" \
        --report-to wandb \
        --wandb-project-name "S-CLIP" \

done
done
done
done
done`

Could you give me some advice?

**Comment by sangwoomo** [comment]
Hi @chirmy, the reported values in the paper are in percentages (%), so they should be multiplied by 100 to make 0.04 * 100 = 4.

**Comment by chirmy** [comment]
Thank you! You have helped me solve the problem that has been bothering me for many days. Thank you very much
